src/main/java/org/commonjava/indy/service/tracking/jaxrs/AdminResource.java

```diff
@@ -156,29 +161,38 @@ public Response getReport(
     @Path( "/{id}/record" )
     @PUT
     public Response initRecord(
-                    @Parameter( description = "User-assigned tracking session key", in = PATH, required = true ) @PathParam( "id" ) final String id,
-                    @Context final UriInfo uriInfo )
+            @Parameter( description = "User-assigned tracking session key", in = PATH, required = true )
+            @PathParam( "id" ) final String id, @Context final UriInfo uriInfo )
     {
         Response.ResponseBuilder rb = Response.created( uriInfo.getRequestUri() );
         return rb.build();
     }
 
-    @Operation( description = "Store record for single tracking content artifact" )
-    @APIResponse( responseCode = "201", description = "Store tracking entry" )
-    @Path( "/report/recordArtifact" )
-    @PUT
-    public Response recordArtifact( final @Context UriInfo uriInfo, final @Context HttpServletRequest request )
```

Comment:
Can we confirm that this PUT method is not used by any other services?

Reply:
@ligangty yes, this was added only for sidecar folo report stuff, no other service used it.

Reply:
Ok. Seems this is a little hacking and strange because it is creating entry through GET API, but as we agreed to use this way, so let's pass it.